libvmi/driver/xen/xen_events.c

```diff
@@ -668,6 +669,9 @@ void xen_events_destroy_new(vmi_instance_t vmi)
     rc = xen->libxcw.xc_monitor_write_ctrlreg(xch, dom, VM_EVENT_X86_CR4, false, false, false);
     rc = xen->libxcw.xc_monitor_write_ctrlreg(xch, dom, VM_EVENT_X86_XCR0, false, false, false);
     rc = xen->libxcw.xc_monitor_software_breakpoint(xch, dom, false);
+    rc2 = xen_set_guest_requested_event(vmi, 0);
```

Comment:
ok

On Wed, Aug 24, 2016 at 10:58 AM, Tamas K Lengyel <notifications@github.com>
wrote:

> In libvmi/driver/xen/xen_events.c
> <https://github.com/libvmi/libvmi/pull/364#discussion_r76073137>:
>
> > @@ -668,6 +669,9 @@ void xen_events_destroy_new(vmi_instance_t vmi)
> >      rc = xen->libxcw.xc_monitor_write_ctrlreg(xch, dom, VM_EVENT_X86_CR4, false, false, false);
> >      rc = xen->libxcw.xc_monitor_write_ctrlreg(xch, dom, VM_EVENT_X86_XCR0, false, false, false);
> >      rc = xen->libxcw.xc_monitor_software_breakpoint(xch, dom, false);
> > +    rc2 = xen_set_guest_requested_event(vmi, 0);
>
> Not really, it's safe to issue these anytime, even if they were not
> enabled previously.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/libvmi/libvmi/pull/364/files/4e2e3eaaec7f4d45ccf48a94cc6c29c8573dc6f1#r76073137>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AA5auOwQOfvd3Kcj1nRIFE5L_3dWXilnks5qjFwZgaJpZM4Jo3PQ>
> .
>